### ISSUE TITLE
Question- How can I create a video from multiple images with time for each frame?

### ISSUE THREAD

**Issue by darkmat13r** [PHP-FFMpeg/PHP-FFMpeg#548]
I want to create a video from multiple images which are stored in a folder.
ffmpeg -r 1/5 -start_number 0 -i "myfolder/img%d.jpg" -c:v libx264 -vf "fps=25,format=yuv420p" output.mp4

this is how i did it with command line. Want to know how can I achieve this with PHP-FFmpeg lib.

**Comment by Mahmood-Hussain** [comment]
is this feature added @jens1o ?

**Comment by jens1o** [comment]
@Mahmood-Hussain No, but you can contribute to this project and then this feature will be available. :)
